### DISCUSSION TITLE
Which is better steel gear or nylon

### DISCUSSION THREAD

**Discussion by Larswagner1987** [danielnilsson9/bbs-fw#248] [Q&A] [answered]
I wanted to install a steel gearbox on my BBSHD and I was told that it is better. Is that true and how loud is it compared to nylon? Thanks for reading.

**Comment by OoohPossum** [comment]
"Better" is relative.

TL;DR: If you want it ways less noisy and don't want to risk damage to many parts of the motor due to steel splinters (two steel gear wheels), stay with nylon gear. If you need to use high currents out of low or no running speed of the motor, the steel wheel is a better choice, if well-greased - but as well noticeably more noisy (quite loud sound) and more prone for secondary damage if one of the wheels get damaged.


The nylon gears of the BBS02 and BBSHD are quite rugged, but don't like spontaneous high resistance.

I.e. very high starting currents out of standing still (this results in very high pulse load to the cogwheel sides) or gear changes resulting in extreme current changes necessary for the motor to keep spinning (while it ran with lower currents beforehand as the gear was more fitting).

As well as long-term loads above 800-950W on the BBS02B and probably 1450-1550W on the BBSHD, which will increase the abrasion to the nylon gear as well.

Had two nylon gears losing teeth by doing wheelies in a gear that wasn't low enough and with a motor configuration set to deliver very high starting currents (almost no ramp). While for common usage it will keep up with the loads, in those situations it is possible to ruin the teeth.

A steel cog wheel / gear will hold ways longer, **IF** greased well and with the right grease. Several greases tend to ruin the plastics (the nylon cog wheel, as example - but that one isn't here anymore after the change, as such other plastic pieces could eventually get softened or damaged).

But: it is ways more noisy, even well-greased. And you will raise the loads to the motor, if you use it all the time on loads that high (over 1500W at a BBSHD).

The BBSHD at least has two thermistors (temperature-resistance-value sensors), one at the MosFET-area and one at the motor.
The BBS02(B) has only one sensor: at the MosFET-area aluminum part. It can be overheated ways easier (motor-side).

If you use your motor all the time with very high loads / currents, the steel gear is a good choice, if well-greased. But if one of the two wheels gets damaged, the splinters will as well cause more damage compared to the steel-nylon combination (where only the nylon gear will get damaged and the splinters are not hard enough to cause that much damage if stuck between the teeth).

**Reply by Larswagner1987** [reply]
Okay, I do more uphill than downhill and I don't know what happened but the motor power is no longer what it was before. My wife has the same bike as me and she is four times faster than me, for whatever reason, I can't understand it or explain what the problem is, and I thought that a metal one would be better because uphill it has 1480 watts, on straight stretches the power then goes back to normal, around 400-600 watts, no more, which is why I wanted to ask whether it would make sense to replace it.

**Reply by OoohPossum** [reply]
On 400-600W you don't need to replace the nylon gear, but with almost 1500W uphill that is a factor for the longevity of the gearing components.

The aspect of "the motor power is no longer what it was before" is a problem, because if the nylon cogwheel / gear is defective (broken teeths basically), you would be able to hear or feel this usually.

If you had an overheated winding in the motor, resulting in a winding short, this would instantly make your motor very inefficient and overheat this piece of winding by even very low currents. As such most likely not the case, it isn't usable anymore, then.

Aside burnt wires, a faulty third phase, a roasted Thermistor limiting the currents, no matter the motor configuration, a semi-faulty controller or mechanical losses - could be a demagnetized permanent magnet array. Due to overheating.

Overheating the magnets results in weakening of their magnetic field intensity. This results in less torque, higher losses and so on. But usually the Thermistor should prevent this.

Would explain less "power" on your bike without mechanical or electronic defects (while mechanical and electronic defects usually lead to more than just "lower power").

But: you can check the status of the gear wheels and the motor - if the nylon gear is damaged somewhere, shows signs of "teeth"losses, then it's a good idea to replace it, as this as well reduces efficiency.


You can test if your battery plays a role with the motor power degradation, as well (by swapping it with the one of your wife), that could be a reason as well.

The nylon gear wheel for itself hardly is able to make you that much slower without showing clear signs (noise, rattling, skipping).

**Reply by Larswagner1987** [reply]
Okay, what I've noticed is that my motor is sluggish, although I've set it higher so that it accelerates quicker. As I said, it's just weird that it works great for my wife and not for me. I could swap the motor by installing this motor in mine and mine in hers and see if that's the problem. We've also noticed with the new FW 1.5, which is new, that our battery runs out quicker, with the 1.4 version. I don't know if that could be the problem. I'll try swapping the motor first and if that doesn't help, I'll try going from 1.5 to 1.4 and see if that's the problem.

**Reply by Larswagner1987** [reply]
@OoohPossum 

I've tried everything and there's no change. The only thing I've noticed is that when braking, the display takes 3-5 seconds to go from 15km/h to 0km/h. I have this on every display of the 4 that I have. What could it be?

**Reply by OoohPossum** [reply]
@Larswagner1987 : Oh, I see - you use different firmware on both bikes? Initially I assumed that by "same bike" you would've used the same firmware version on both bikes, as well.

The software could be responsible for motor behavioral changes, definitely.
But I am not specifically a coder, just hardware specialist. Regarding the software others can surely help you better.

Regarding "tried everything": As such you swapped the motor to the bike of your wife and now her bike has the same lower power problem that the motor had initially with your setup (wires, battery, etc.) on the setup of your wife?

That is then linked to the motor itself or its firmware / configuration, surely. If you already checked if it would behave different with the FW 1.4 and it didn't change (even with same settings as your wife's motor on v. 1.4), then that limits the options to the motor and controller, physically. As such: a fault or defect on one of their components.

**Reply by Larswagner1987** [reply]
@OoohPossum 

My wife and I have version 1.4, everything is installed the same and the settings are the same, the only thing I noticed is that my e-bike display is about 5 seconds slow, it goes from 15 to zero when the brake is pulled, now the question is, is it the sensor on the frame or the magnet or the software.

**Reply by OoohPossum** [reply, marked as answer]
Regarding the display: If it takes 3-5 seconds to go from 15 to 0 km/h, but only on your motor (while you flashed it back to 1.4?), it directs the analysis to the motor and controller, as well.

If the software was flashed (downgraded) without fault and you set up working configuration (like the one on your wife's bike), then there are not that much possibilities left.

The slow lowering of km/h on the display (if you tested multiple of them) could link to the speed sensor or its motor cable, a controller fault, a hardware fault of the motor itself ..

If you flashed 1.4 back to it, that's at least excluding a flashing error on the previous 1.5 flash from our analysis. I hope you set the frequency right before flashing, but if you did this process multiple times in the past and it worked, I don't think that's the cause, as well.


Edit: Ah, you answered in the meantime. Okay, that is some new information.

Same settings, same flashing of v. 1.4 - excludes this as the cause, as well (if the flashing procedures were without fault, at least, but that is something you could check easily by re-flashing it).

The magnet couldn't cause a "slow lowering", as such sensor, cable (including motor cable to the sensor) or controller.
But the other question is: does it play a role in the much lower power on your motor?

If there's a controller defect, that could result in multiple symptoms, but it will be hard to analyze the exact component of the motor (motor, wiring, controller, hall sensor is a possibility as well) without replacement parts and the possibility to try them out.


A slower lowering of the km/h is linked to the speed sensor signal, at least. You can try to stop the wheel with sensor and look at how long the display takes to notice it, but it should never be more than 1 second around. 3-5 seconds is ways too high.

**Reply by Larswagner1987** [reply]
I found out that it is the brake on the rear wheel, which slows the wheel down slightly. If I take it off and the wheel turns then it shows correctly, but that is embarrassing now 😂, it can happen any time, but you can't hear any grinding and it is properly centered, which is strange, thanks for the help.

**Reply by OoohPossum** [reply]
Oh, that's an interesting cause for the slow display - tho, very understandable regarding the other symptoms.

No problem, I hope that your motor works like expected after the grinding brake is fixed.

**Comment by EigenMania** [comment]
Steel will take much more abuse compared to nylon, but will be quite louder. 

I could find one comparison video online, but they compare nylon to aluminium to brass: https://www.youtube.com/watch?v=s6LjyQhM0jI

Still, any metal will be much louder than nylon. 

An in-between option would be a PEEK gear. They are tricky to find online: 

- https://highvoltagekits.com/hv-shop/p/peek-gear-for-bbshd-order-via-email-or-discord-not-sold-out
- https://lunacycle.com/bbshd-silent-peek-gear-nylon-gear-replacement/

(Luna claims that carbon composite mixed with the PEEK makes it even stronger). 

Steel is overkill unless you really want to beat your bike from a standstill (ie pull wheelies or take off full throttle in the highest gear on an uphill) or swap out the stock BBSHD controller for something that pushes 3000+ watts.

**Reply by OoohPossum** [reply]
The PEEK-CF replacement gear wheel is a good alternative to the nylon original gear, but unfortunately either hard to get the hands on (limited stock, Luna hasn't it on stock since a while, last as I checked) - or very pricey.

For Germany, Austria and so on it is quite easy to get over fasterbikes (here: www.fasterbikes.eu/en/parts/bafang-bbshd-1000w/392/bafang-bbshd-peek-reduction-gear-nylon-replacement ) - but for a very high price.

It is not as rugged as a brass or steel gear, as well not great at self-lubrication (brass is top class in this area), but very silent in comparison to the metal gears, that's right.

**Comment by WvBijsterveld** [comment]
just wanna weigh in on the conversation, replacing the nylon gear with a steel or even a PEEK one wouldn’t be worth it for you lars.

I have run the nylon gear a year straight and i’m running 3800+ watts, i have a peek gear laying around that i have yet to switch in.

regarding the 400-600 watt issue you have is something i’ve read about a lot on these issues so i guess it’s just a rare motor issue.

the display being slow from 15km/h to 0 could be just that is it waiting for no signal to happen and it is based on second intervals, this is pretty common as it is waiting for the sensor to detect something before declaring 0 speed. 

Best bet would be to increase magnets in wheel so you have way better low speed accuracy.

**Reply by Larswagner1987** [reply]
Hello , thank you, so if I use two magnets, should I put the second one opposite, not at the same height, i.e. if one magnet is at the top, should I put the other one at the bottom?

**Reply by OoohPossum** [reply]
Two magnets or even three, four.. (I use two myself, that's enough) have to be located as such that they are in the same position, near the sensor, going in the same direction. But on opposite "end" (turning the wheel half a full turn, then placing it where the other sensor is - just 180° wheel-turn shifted) of the wheel.

No need for different magnetic orientation or such. As one magnet with right orientation will trigger basically both directions by passing the sensor (On-Off) and that's a necessity for a full signal. Two magnets then do this twice, you have to set the amount of signals to 2 in configuration tool, else the displayed speed is double the real speed.

Aside from this no other changes are necessary, but remember that the physical sensor and controller only can calculate and analyze speeds up to a specific limit which seems to be given by the signal amount. Should be no hassle in your case, as if you go uphill, you don't break 60 km/h usually (I had the wrong speeds occurring at above 60 - 75 km/h with two magnets).

@WvBijsterveld : That's interesting, 3800 Watts and no decaying of the nylon gear, yet. Impressive. Sounds like you use the power at higher rounds per minute (on the nylon gear) and not from start-up in high gears?

The faster the gear wheels can turn at a given current, the lower is the torque on same total power output (plain physics), that can save the nylon wheel from damage. I only used 1800-1900W peak, but happened to damage the nylon wheel due to almost zero rounds per minute and high torque on the teeth.

**Reply by WvBijsterveld** [reply]
I have moments where i have to use more power at lower nylon gear speeds, but most of the time i go full throttle above like 30+ km/h.
With my controller i have a very high motor max rpm so the temps are actually better then stock, because i can use my low gears from 0 to 70 km/h without shifting once. however i should note i have like a set motor rpm which equals to full power gain and its like a middle point, anything before or after it reduces max power. so if i full throttle at low rpm im not giving it 3800 watts but i believe under 15km/h in high gear it not more then +/- 2200 watts.

there are exceptions, i have ran 52, 56 and once a 60 tooth front chainring to try and get a personal top speed, and in this scenario you're putting so much stress even when using the 36t back speed.

I've certainly put a lot of stress on her but so far, luckily, nothing has yet to break in the motor.
